backend/dataall/modules/metadata_forms/services/metadata_form_service.py

```diff
@@ -96,8 +135,9 @@ def get_metadata_form_by_uri(uri):
     @MetadataFormAccessService.can_perform('DELETE')
     def delete_metadata_form_by_uri(uri):
         mf = MetadataFormService.get_metadata_form_by_uri(uri)
-        with get_context().db_engine.scoped_session() as session:
-            return session.delete(mf)
+        if mf:
```

Comment:
nit: consolidate to
```
if mf := MetadataFormService.get_metadata_form_by_uri(uri):
    with ...
```

Reply:
[8bbefd2](https://github.com/data-dot-all/dataall/pull/1480/commits/8bbefd22389d62a137fe26de245f7f2bbea15a7a)